Order.Management.PR/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Order.Management
+{
+    class Program
+    {
+        // Main entry
+        static void Main(string[] args)
+        {
+            var (customerName, address, dueDate) = CustomerInfoInput();
+
+            var orderedShapes = CustomerOrderInput();
+
+            InvoiceReport(customerName, address, dueDate, orderedShapes);
+
+            CuttingListReport(customerName, address, dueDate, orderedShapes);
+
+            PaintingReport(customerName, address, dueDate, orderedShapes);
+        }
+        
+        // Order Circle Input
+        public static Circle OrderCirclesInput()
```

Comment:
`OrderCirclesInput`, `OrderSquaresInput` and `OrderTrianglesInput` are quite similar, we could abstract it to a generic method. example `T OrderShapesInput<T>() where T: Shape`.
Another suggestion is moving this logic to a separate factory class. It will help us to separate the logic out and make it testable.